tests/test_trainer.py

```diff
@@ -44,24 +47,6 @@
 PATH_SAMPLE_TEXT = f"{get_tests_dir()}/fixtures/sample_text.txt"
 
 
-class RegressionDataset:
```

Comment:
I am going to insist one more time. Please do not delete the existing code for `RegressionDataset`, we need to be able to reuse this even if `is_torch_available()` is false.

Reply:
Sure.